civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -854,12 +854,100 @@ function civihr_employee_portal_views_default_views() {
 
     if (isset($view)) {
       $views[$view->name] = $view;
+      civihr_employee_portal_fix_custom_fields_mapping($view);
     }
   }
 
   return $views;
 }
 
+/**
+ * To fix mapping on Views import of CiviCRM custom fields
+ *
+ * @param view $view
+ *     the view that is being imported / recreated
+ */
+function civihr_employee_portal_fix_custom_fields_mapping($view) {
+  global $databases;
+  // iterating displays in view
+  foreach ($view->display as $display) {
+    // iterating options on display: fields, filters, arguments
+    foreach ($display->display_options as &$display_option) {
+      if (!is_array($display_option)) {
+        continue;
+      }
+      foreach ($display_option as &$option) {
+        // only continue if this option is about a field
+        if (!isset($option['field'])) {
+          continue;
+        }
+        $option_id_parts = explode('_', $option['field']);
+        $final_id_part = array_pop($option_id_parts);
+        // it must contain underscores and a number at the end to be checked
+        if (!$option_id_parts > 0 || !intval($final_id_part)) {
```

Comment:
`$option_id_parts` is an array. I don't think it makes sense to compare if it is greater than a number. Also, why are you negating the array? I think that what you've meant here is `count($option_id_parts) === 0)` or `empty($option_id_parts)`.